### PR TITLE
Feature/obssub solnlist

### PR DESCRIPTION
**Pull request by borkarsaish65**

### CODE REVIEW

File: module/observationSubmissions/helper.js [hunk 1]
```diff
@@ -1000,13 +1010,19 @@ module.exports = class ObservationSubmissionsHelper {
           entityIds.push(submission.entityId);
         });
 
-        let entitiesData = await entitiesHelper.entityDocuments(
+        let entitiesDocumentAPICall = await entityManagementService.entityDocuments(
```
Comment (VISHNUDAS-tunerlabs) [inline module/observationSubmissions/helper.js:1013]: @borkarsaish65 rename this to entitiesDetails

**Review by VISHNUDAS-tunerlabs — CHANGES_REQUESTED** [review]

File: module/observationSubmissions/helper.js (hunk 1 above)
Comment (borkarsaish65) [reply module/observationSubmissions/helper.js:1013]: done

File: controllers/v1/observationsController.js [hunk 2]
```diff
@@ -843,6 +850,8 @@ module.exports = class Observations extends Abstract {
           status: 'started',
         };
 
+        console.log(submissionDocument,'submissionDocument')
```
Comment (VISHNUDAS-tunerlabs) [inline controllers/v1/observationsController.js]: @borkarsaish65 please remove console

File: module/observations/helper.js [hunk 3]
```diff
@@ -403,6 +403,7 @@ module.exports = class ObservationsHelper {
 
           let observationEntitySubmissions = {};
           submissions.forEach((observationEntitySubmission) => {
+            console.log(observationEntitySubmission,'<--target')
```
Comment (VISHNUDAS-tunerlabs) [inline module/observations/helper.js]: @borkarsaish65 please remove console

File: module/observationSubmissions/helper.js (hunk 1 above)
Comment (VISHNUDAS-tunerlabs) [reply module/observationSubmissions/helper.js:1012]: @borkarsaish65 please change every where it is used

**Review by VISHNUDAS-tunerlabs — COMMENTED** [review]
reviewed on 24 Oct 5:20

File: controllers/v1/observationsController.js (hunk 2 above)
Comment (borkarsaish65) [reply controllers/v1/observationsController.js]: done

File: module/observations/helper.js (hunk 3 above)
Comment (borkarsaish65) [reply module/observations/helper.js]: done

File: module/observationSubmissions/helper.js [hunk 4]
```diff
@@ -1000,13 +1009,19 @@ module.exports = class ObservationSubmissionsHelper {
...
-        let entitiesData = await entitiesHelper.entityDocuments(
+        let entitiesDetails = await entityManagementService.entityDocuments(
           {
             _id: { $in: entityIds },
           },
           ['metaInformation.externalId', 'metaInformation.name'],
         );
 
+        if(!entitiesDetails.success){
+          throw new Error("Entity Management Service down.")
```
Comment (VISHNUDAS-tunerlabs) [inline module/observationSubmissions/helper.js:1020]: @borkarsaish65 define this message as a response please

**Review by VISHNUDAS-tunerlabs — CHANGES_REQUESTED** [review]
reviewed on 24 Oct 5:20

File: module/observationSubmissions/helper.js (hunk 4 above)
Comment (borkarsaish65) [reply module/observationSubmissions/helper.js]: done

File: module/observationSubmissions/helper.js (hunk 1 above)
Comment (borkarsaish65) [reply module/observationSubmissions/helper.js]: done

**Review by VISHNUDAS-tunerlabs — APPROVED** [review]

**Pull request merged**